core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -254,6 +254,17 @@ public ClientFactoryBuilder tlsNoVerify() {
         return this;
     }
 
+    /**
+     * Disables the verification of server's key certificate chain for specific hosts.
+     * <strong>Note:</strong> You should never use this in production but only for a testing purpose.
+     *
+     * @see #tlsCustomizer(Consumer)
+     */
+    public ClientFactoryBuilder tlsNoVerifyHosts(String... insecureHosts) {
+        tlsCustomizer(b -> b.trustManager(IgnoreHostsTrustManager.of(insecureHosts)));
+        return this;
+    }
```

Comment:
A user might want to do the following:

```java
cfb.tlsNoVerifyHosts("a.com");
cfb.tlsNoVerifyHosts("b.com");
...
```

How about keeping the list of the insecure hosts and setting the trust manager when the `ClientFactory` is built? We will also have to make sure `tlsNoVerify()` and `tlsNoVerifyHosts()` are mutually exclusive, throwing an `IllegalStateException` (or should we make `tlsNoVerifyHost()` always win?)

Reply:
I agree with making them exclusive. Makes it less ambiguous about which hosts are being disabled (or all of them).